docs/source/federation-versions.mdx

```diff
@@ -21,6 +21,134 @@ This article describes notable changes and additions introduced in each minor ve
 
 - **If you maintain a [subgraph-compatible library](./building-supergraphs/supported-subgraphs/),** consult this article to stay current with recently added directives. All of these directive definitions are also listed in the [subgraph specification](./subgraph-spec/#subgraph-schema-additions).
 
+## v2.5
+
+<hr/>
+
+<CodeColumns cols="3">
+
+<div>
+
+First release
+
+**July 2023**
+
+</div>
+
+<div>
+
+Available in GraphOS?
+
+**No**
+
+</div>
+
+<div>
+
+Minimum router version
+
+**`TBD`**
+
+</div>
+
+</CodeColumns>
+
+<hr/>
+
+#### Directive changes
+
+<table>
+  <thead>
+    <tr>
+      <th>Topic</th>
+      <th>Description</th>
+    </tr>
+  </thead>
+
+<tbody>
+<tr>
+<td>
+
+##### `@authenticated`
+
+</td>
+<td>
+
+> ⚠️ **This is an [Enterprise feature](/router/enterprise-features) of the Apollo Router.** It requires an organization with a [GraphOS Enterprise plan](https://www.apollographql.com/pricing/).</br>
```

Comment:
```suggestion
```

We can remove these from the table to avoid clutter. (But obviously keep them in other docs.)